components/Header/Header.module.scss

```diff
@@ -23,36 +24,36 @@
 }
 
 .logo {
-  align-items: center;
-  border: none;
   display: flex;
+  align-items: center;
   margin-right: 1rem;
+  border: none;
 
   .wordmark {
     display: none;
   }
 
   .monogram {
-    width: 35px;
+    width: 2.1875rem;
```

Comment:
Is there a reason you converted many `px` values to `rem`? There's very specific places where `px` are preferred, namely around graphic sizing which, in this example, is the case. I'd prefer to see `px` to remain in `px` unless it directly affects font-size.